docs/self-managed/operational-guides/update-guide/860-to-870.md

```diff
@@ -59,3 +59,73 @@ In that case, we append the following records:
 - `ASSIGN`
 - `ASSIGNING`
 - `ASSIGNED`
+
+## Prefix migration
+
+:::warning
+It is a requirement that Operate and Tasklist (for example, using `camunda.operate.elasticsearch.indexPrefix`) have
+custom prefixes defined and that you are migrated to another prefix.
+:::
+
+### Make the Elasticsearch instance reachable
+
+The migration relies on communicating directly with Elasticsearch.
+
+The below example relies on a kubernetes environment but any approach works as long as the Elasticsearch instance is reachable
+on `http://localhost:9200`.
+
+- **Kubernetes**: Port-forward your Elasticsearch instance to ensure it is reachable. In the following example, `camunda-platform` can be replaced by the service name:
+
+  ```shell
+  kubectl port-forward svc/camunda-platform-elasticsearch 9200:9200
+  ```
+
+### Create harmonized indices
+
+To begin creating the harmonized indices for migration, first pull the following Docker image:
+
+```shell
+docker pull camunda/zeebe:SNAPSHOT
+```
+
+Run the command below, replacing `CAMUNDA_DATABASE_PREFIX` value with your desired prefix:
+
+```shell
+docker run --rm -it --env CAMUNDA_DATABASE_URL="host.docker.internal:9200" --env CAMUNDA_DATABASE_INDEXPREFIX=<prefix> --entrypoint /usr/local/zeebe/bin/schema camunda/zeebe:SNAPSHOT
+```
+
+### Migrating indices
+
+The migration of the indices assumes that elasticsearch is reachable on
+localhost:9200.
+
+Execute the `.migrate.sh` bash script which has dependencies on `jq` and `curl`.
+It is required that the: operate, tasklist and new prefix are defined.
```

Comment:
```suggestion
We better UX we provide a bash script: `.migrate.sh` to run the migration. The bash script which has dependencies on `jq` and `curl`.
It is required that the: operate, tasklist and new prefix are defined.

If you would like to run the migration on your own, please have a look at the script, what steps are necessary.
```

Reply:
Actually, can you bring back the description from before? I would like to have it still, the bash is just a good add-on to make it easier, but I think it would be useful to understand what is happening and some might want to ran it manually.

Reply:
We might want to also add that they should make sure to backup their data before plus testing the script first in dev environment to be safe.